### DISCUSSION TITLE
Maintainership of rust-bitcoinconsensus

### DISCUSSION THREAD

**Discussion by tcharding** [rust-bitcoin/rust-bitcoin#991] [General]
Prodding folks again.

The `rust-bitcoinconsensus` crate is without maintainership. There are open PRs there for 4 months. 

I have attempted to gently prod folks already by email and here in this crate. FTR @apoelstra did respond to prodding confirming he would start watching the repo and @dr-orlovsky did respond to prodding stating that he was watching but was not a maintainer. (@stevenroose has told me offline that he is totally snowed under by GitHub notifications and more or less doesn't get them.)

What are we going to do about that? Is anyone able to please review, ACK, or NACK?

@apoelstra , if no-one objects, perhaps I could please be made a maintainer of that crate?

Thanks

 me\ backs quietly out of room

**Comment by dr-orlovsky** [comment]
As I expressed in my letter to rust-bitcoin maintainers already, I am not going to regularly contribute or maintain any project from which I can be suspended or  expelled in a random manner when somebody would not like my conduct (since I am not going to comply with - or meet anybody's expectations on any conduct, or even take them into an account). So guarantees of no-CoC (or an explicit CoC clearly stating that non-technical conduct can't be used as a base for suspending or stopping mainterership) are required for me to contribute to any of rust-bitcoin projects in a future.

**Reply by tcharding** [reply]
No worries, https://github.com/rust-bitcoin/rust-bitcoin/pull/992

**Comment by apoelstra** [comment]
@tcharding I would really appreciate you taking on maintainership of this crate. I will add you.
